NDcPP_v2_2e.adoc

```diff
@@ -2636,52 +2636,54 @@ If TLS is selected in FPT_ITT, then certificate validity is tested in accordance
 
-[arabic, start=107]
-. {blank}
+*_Application Note {counter:appnote_count}_*
+
 
 If ciphersuites with elliptic curves were selected in FCS_TLSC_EXT.1.1, a selection of one or more curves is required. If no ciphersuites with elliptic curves were selected in FCS_TLSC_EXT.1.1, then “not present the Support Elliptic Curves/Supported Groups Extension” should be selected.
 
 This requirement limits the elliptic curves allowed for authentication and key agreement to the NIST curves from FCS_COP.1/SigGen and FCS_CKM.1 and FCS_CKM.2. This extension is required for clients supporting Elliptic Curve ciphersuites.
 
 If ciphersuites with DHE key agreement were selected FCS_TLSC_EXT.1.1 and the TOE supports TLS FFC groups (e.g. ffdhe2048), this extension is required. This extension is not required if the TOE only supports non-TLS FFC groups (e.g. Group 14).
 
-*FCS_TLSS_EXT.1 TLS Server Protocol Without Mutual Authentication*
+*FCS_TLSS_EXT.1 TLS Server Protocol*
 
-*FCS_TLSS_EXT.1.1* The TSF shall implement [selection: __TLS 1.2 (RFC 5246), TLS 1.1 (RFC 4346)__[.underline]##]## and reject all other TLS and SSL versions. The TLS implementation will support the following ciphersuites:
+*FCS_TLSS_EXT.1.1* The TSF shall implement [selection: _TLS 1.3 (RFC 8446), TLS 1.2 (RFC 5246)_] and [selection: _TLS 1.1 (RFC 4346), no additional TLS version_] and reject all other TLS and SSL versions. The TLS implementation will support the following ciphersuites:
 
-____
-[selection:
+_[selection:_
 
-[.underline]#_select supported ciphersuites from List 1_] _and no other ciphersuites_#.
-____
+* _select supported ciphersuites for TLSv1.1 and/or TLSv1.2 from List 1_
 
-[arabic, start=108]
-. {blank}
+* _select supported ciphersuites for TLSv1.3 from List 2_
 
-The ciphersuites to be tested in the evaluated configuration are limited by this requirement and must be selected from the ciphersuites defined in List 1. The ST author should select the optional ciphersuites that are supported. Even though RFC 5246 mandates implementation of specific ciphers, there is no requirement to implement TLS_RSA_WITH_AES_128_CBC_SHA in order to claim conformance to this cPP.
+_]_ and no other ciphersuites.
 
-These requirements will be revisited as new TLS versions are standardized by the IETF.
+
+*_Application Note {counter:appnote_count}_*
+
+
+_The ciphersuites to be tested in the evaluated configuration are limited by this requirement and must be selected from the ciphersuites defined in List 1. The ST author should select the optional ciphersuites that are supported. Even though RFC 5246 mandates implementation of specific ciphers, this cPP depreciates use of SHA-1. Support for TLS_RSA_WITH_AES_128_CBC_SHA is not claimed in order to claim conformance to this cPP._
+
+
+_These requirements will be revisited as new TLS versions are standardized by the IETF._
 
 _In a future version of this cPP TLS v1.2 will be required for all TOEs._
```

Comment:
@qtrinh3 Application note 124 in the NDcPP TLSWGProposal_TLSv13_v13d MS Word document is missing from your edits.